app/services/planning_application_creation_service.rb

```diff
@@ -14,6 +14,7 @@ def initialize(planning_application: nil, **options)
       @local_authority = planning_application.local_authority
       @api_user = planning_application.api_user
       @send_email = false
+      @session_id = SecureRandom.uuid
```

Comment:
Just generating a new session_id when a planning application is cloned. Can't use the same one due to the unique constraint and we don't care about the session_id for cloned applications

Reply:
Maybe make the column nullable? That way you can have the unique index but don't have fake session ids floating around.